btk/draw_blends.py

```diff
@@ -617,9 +617,9 @@ def render_single(self, entry, filt, psf, survey, extra_data):
         # get galaxy flux
         try:
             mag_name = f"{survey.name}_{filt.name}"
-            gal_flux = mag2counts(entry[mag_name], survey.name, filt.name).to("electron").value
+            gal_flux = mag2counts(entry[mag_name], survey, filt).to("electron").value
```

Comment:
```suggestion
            gal_flux = mag2counts(entry[mag_name], survey, filt).to_value("electron")
```

Reply:
good catch! For some reason in my mind I read your post as two separate operations 😆